BMM.Core/Constants/ImageServiceConstants.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+
+namespace BMM.Core.Constants
+{
+    public class ImageServiceConstants
+    {
+        public const string ImageCacheFolder = "img";
+        public const int ImageCacheMemorySize = 100 * 1024 * 1024; //100 MB
+
+        public static readonly TimeSpan DiskCacheDuration = TimeSpan.FromDays(7);
```

Comment:
why not 30 days?

Reply:
the url changes when a new version is available. that should allow us to cache it for a long time

Reply:
Changed to 30 days